Added support for supported languages

koshtinikhilesh opened the pull request:
## Fixes # by...

### Changes proposed in this pull request:

* ...
* ...

### Status

- [ ] READY
- [ ] HOLD
- [X] WIP (Work-In-Progress)

### How to verify this change

*Fill out this section so that a reviewer can know how to verify your change.*

### Additional notes

*If applicable, explain the rationale behind your change.*

koshtinikhilesh commented:
Removed dependancy of CONVERTED_CLASSES for listing supported languages.

koshtinikhilesh commented:
Can you please have a look for review or merge this PR?

erozqba commented:
@koshtinikhilesh the merge is blocked if the test coverage decreases, could you please add tests for this functionality?

koshtinikhilesh commented:
> @koshtinikhilesh the merge is blocked if the test coverage decreases, could you please add tests for this functionality?

Yes, sure. Sorry for the late reply.
I have added tests for the new functionality. and fixed the failures.
